docs/developing.rst

```diff
@@ -4,43 +4,18 @@ Release Checklist
 
 This page describes the process of releasing new versions of Planemo.
 
-This release checklist is based on the `Pocoo Release Management Workflow
-<http://www.pocoo.org/internal/release-management/>`_.
-
-This assumes ``~/.pypirc`` file exists with the following fields (variations)
-are fine.
-
-::
-
-    [distutils]
-    index-servers =
-        pypi
-        test
-    
-    [pypi]
-    username:<username>
-    password:<password>
-    
-    [test]
-    repository:https://testpypi.python.org/pypi
-    username:<username>
-    password:<password>
-
-
 * Review ``git status`` for missing files.
 * Verify the latest Travis CI builds pass.
+* Update ``HISTORY.rst`` with the help of ``scripts/bootstrap_history.py``
 * ``make open-docs`` and review changelog.
 * Ensure the target release is set correctly in ``planemo/__init__.py`` (
   ``version`` will be a ``devN`` variant of target release).
 * ``make clean && make lint && make test``
-* ``make release``
-
-  This process will push packages to test PyPI, allow review, publish
-  to production PyPI, tag the git repository, push the tag upstream,
-  and modify the Homebrew recipe. If changes are needed, such as manual
-  changes to the homebrew recipe, this can be broken down into steps 
-  such as:
-
-  * ``make release-local``
-  * ``make push-release``
-  * ``make release-brew``
+* Commit outstanding changes.
+* Update version and history, commit, add tag, mint a new version and push
+  everything upstream with ``make release``
+* The new tag should automatically push the new release to PyPI via the
+  ``deploy`` job of the GitHub Actions workflow defined in
+  ``.github/workflows/ci.yaml`` .
+  If this didn't work, you can ``git checkout`` the tag and push to PyPI by
+  executing ``make release-artifacts``
```

Comment:
Awesome sauce, thanks for clarifying the process and how handle failures even!

Reply:
Well, I'm not sure the upload will work the first time, so better plan ahead :laughing: